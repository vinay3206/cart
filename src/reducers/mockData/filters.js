const filters = [
  {
    category: 'Brands',
    id: 10001,
    options: [
      {
        label: 'Mangolia',
        id: 1,
      },
      {
        label: 'Meiji',
        id: 2,
      }
    ],
  },
  {
    category: 'Price',
    id: 10002,
    options: [
      {
        label: '0-5',
        id: 3,
        min: 0,
        max: 5,
      },
      {
        label: '5-10',
        id: 4,
        min: 5,
        max: 10,
      }
    ],
  }
];

export default filters;
