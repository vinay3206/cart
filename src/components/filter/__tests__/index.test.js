import React from 'react';
import renderer from 'react-test-renderer';

import Filter from '../index';
import filters from '../../../reducers/mockData/filters';

it('renders correctly', () => {
  const component = <Filter
    filters={filters}
    onFilterChange={() => {}}
  />;
  const tree = renderer
    .create(component)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
