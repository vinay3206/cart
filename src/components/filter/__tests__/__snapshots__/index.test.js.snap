// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="filter"
  >
    <span
      className="filter-category"
    >
      Brands
    </span>
    <div
      className="filter-option"
    >
      <input
        checked={undefined}
        onChange={[Function]}
        type="checkbox"
      />
      <span
        className="filter-option-label"
      >
        Mangolia
      </span>
    </div>
    <div
      className="filter-option"
    >
      <input
        checked={undefined}
        onChange={[Function]}
        type="checkbox"
      />
      <span
        className="filter-option-label"
      >
        Meiji
      </span>
    </div>
  </div>
  <div
    className="filter"
  >
    <span
      className="filter-category"
    >
      Price
    </span>
    <div
      className="filter-option"
    >
      <input
        checked={undefined}
        onChange={[Function]}
        type="checkbox"
      />
      <span
        className="filter-option-label"
      >
        0-5
      </span>
    </div>
    <div
      className="filter-option"
    >
      <input
        checked={undefined}
        onChange={[Function]}
        type="checkbox"
      />
      <span
        className="filter-option-label"
      >
        5-10
      </span>
    </div>
  </div>
</div>
`;
